torchvision/models/detection/_utils.py

```diff
@@ -470,7 +470,12 @@ def retrieve_out_channels(model: nn.Module, size: Tuple[int, int]) -> List[int]:
     return out_channels
 
 
-def _topk_min(input: Tensor, orig_kval: int, axis: int) -> Tensor:
+@torch.jit.unused
+def _fake_cast_onnx(v: Tensor) -> int:
+    return v
```

Comment:
mypy error:
```
torchvision/models/detection/_utils.py:475: error: Incompatible return value
type (got "Tensor", expected "int")  [return-value]
        return v
```

Reply:
you need to turn off mypy. This proposed patch is taken from the following file:
https://github.com/pytorch/vision/blob/93c85bbcc31f8d5a052daf06f2f91f39697af1a4/torchvision/models/detection/transform.py#L19-L22

And it is not being scanned by mypy:
https://github.com/pytorch/vision/blob/93c85bbcc31f8d5a052daf06f2f91f39697af1a4/mypy.ini#L41-L43

If you turn off the warnings, it should work.